packages/server/src/fhir/repo.ts

```diff
@@ -2344,6 +2347,15 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
       } finally {
         this.endTransaction();
       }
+
+      const baseDelayMs = config.transactionExpBackoffBaseDelayMs ?? defaultExpBackoffBaseDelayMs;
+      // Attempts are 0-indexed, so first wait after first attempt will be somewhere between baseDelayMs 75% and 100%
```

Comment:
Is delaying the first retry here intentional?  Previously, an immediate retry was generally quite effective in getting a successful outcome

Reply:
Yeah I think as we scale we may notice the immediate retry might not be as effective, especially in cases where there is a large volume of individual requests that conflict with each other (eg. hundreds per second). I think in the case where we already have contention this small penalty is not much overhead and should help the overall health of the system since we get some jitter baked in which may help to smooth bursty, contentious traffic. We can always downtune the base delay if that seems too high too.